Mathport/Syntax/Transform/Tactic.lean

```diff
@@ -84,24 +83,20 @@ mathport_rules
 mathport_rules
   | `(show $ty:term from by $seq:tacticSeq) =>
     `(show $ty:term by $seq:tacticSeq)
-  | `(suffices $ty:term from by $seq:tacticSeq
-      $t) =>
+  | `(suffices $ty:term from by $seq:tacticSeq; $t:term) =>
     `(suffices $ty:term by $seq:tacticSeq
       $t)
   | `(tactic| suffices $ty:term from by $seq:tacticSeq) =>
     `(tactic| suffices $ty:term by $seq:tacticSeq)
 
 -- push `by` before `have`, `let`, `suffices` so that it can be formatted at the end of a line
 mathport_rules
-  | `(have $hd:haveDecl
-      by $[$seq:tactic]*) =>
+  | `(have $hd:haveDecl; by $[$seq:tactic]*) =>
```

Comment:
Note: this now matches both the semicolon- as well as the newline-separated variant.